temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
Avoid using print statements for logging in production code. Use a logging framework instead to handle different logging levels and outputs.